Docker Script is Broken

The Docker run script https://github.com/substrate-developer-hub/substrate-node-template/blob/master/scripts/docker_run.sh is suffering from a problem with the Rust nightly https://github.com/substrate-developer-hub/substrate-node-template/issues/90. It should use the nightly specified by the `rust-toolchain` file.

As reported by @anhcao142 https://github.com/substrate-developer-hub/substrate-node-template/issues/85#issuecomment-710107030

@TriplEight I updated the toolchain file to `nightly-2020-09-20`, which is what I found by running the command you specified, but I'm still getting more or less the same error.

```sh
     Compiling substrate-wasm-builder v2.0.1
     Compiling wasm-build-runner-impl v1.0.0 (/var/www/node-template/target/release/wbuild-runner/node-template-runtime1437524645330841452)
      Finished release [optimized] target(s) in 3m 36s
       Running `/var/www/node-template/target/release/wbuild-runner/node-template-runtime1437524645330841452/target/x86_64-unknown-linux-gnu/release/wasm-build-runner-impl`
  Rust WASM toolchain not installed, please install it!

  Further error information:
  ------------------------------------------------------------
     Compiling wasm-test v1.0.0 (/tmp/.tmpZIy6bF)
  error[E0463]: can't find crate for `std`
    |
    = note: the `wasm32-unknown-unknown` target may not be installed

  error: aborting due to previous error

  For more information about this error, try `rustc --explain E0463`.
  error: could not compile `wasm-test`

  To learn more, run the command again with --verbose.
  ------------------------------------------------------------

warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/rust-toolchain
+++ b/rust-toolchain
@@ -1,1 +1,2 @@
-nightly-2020-10-06
+stable-x86_64-unknown-linux-gnu
+nightly-2020-10-05

